public/locales/ru/translation.json

```diff
@@ -1,33 +1,39 @@
 {
-  "auth": {
-    "fieldLabels": {
-      "password":"Пароль",
-      "email":"Email",
-      "otp":"OTP"
+    "auth": {
+        "fieldLabels": {
+            "password": "Пароль",
+            "email": "Email",
+            "otp": "OTP"
+        },
+        "fieldErrors": {
+            "emailRequired": "Email обязателен",
+            "emailGeneric": "Некорректный Email",
+            "passwordMinLength": "Пароль должен содержать минимум 6 символов",
+            "otpLength": "OTP должен состоять из 6 символов"
+        },
+        "login": "Вход",
+        "signup": "Регистрация",
+        "verifyMail": "Подверждение Email"
     },
-    "fieldErrors": {
-      "emailRequired":"Email обязателен",
-      "emailGeneric":"Некорректный Email",
-      "passwordMinLength":"Пароль должен содержать минимум 6 символов",
-      "otpLength":"OTP должен состоять из 6 символов"
-    },
-    "login": "Вход",
-    "signup": "Регистрация",
-    "verifyMail": "Подверждение Email"
-  },
-
-  "submit":"Подтвердить",
-  
-  "Welcome Back": "С Возвращением",
-
-  "add": "Добавить",
-  "delete": "Удалить",
-  "update":"Обновить",
-  "search": "Поиск",
-  "cancel":"Отмена",
-  "confirm":"Подтвердить",
-
-  "department": "отдел",
-
-  "skill": "навык"
+    "submit": "Подтвердить",
+    "Welcome Back": "С Возвращением",
+    "add": "Добавить",
+    "delete": "Удалить",
+    "update": "Обновить",
+    "search": "Поиск",
+    "cancel": "Отмена",
+    "confirm": "Подтвердить",
+    "department": "отдел",
+    "skill": "навык",
+    "Profile": "Профиль",
+    "Settings": "Настройки",
+    "Logout": "Выйти",
+    "Home": "Главная",
+    "Employees": "Сотрудники",
+    "Projects": "Проекты",
+    "CVs": "Резюме",
+    "Departments": "Отделы",
+    "Positions": "Должности",
+    "Skills": "Навыки",
+    "Languages": "Языки"
```

Comment:
Some of the words are capitalized because I thought it would be better practice to retrieve a word with proper capitalizing instead of capitalizing it later in code if we need a capitalized word. I also checked how other interns did it and some of them did it just the same. If it's better practice to create a capitalizing function to import it every time we need to call it on {t("...")} later then I'll do it.